### DISCUSSION TITLE
Is there any detailed example for how to build a simple dashboard with different menus or tabs?

### DISCUSSION THREAD

**Discussion by qdriven** [illacloud/illa-builder#1229] [General]
Is there any detailed example for how to build a simple dashboard with different menus or tabs?

still hard to start a simple application.

**Comment by wenzhuo2022** [comment]
Hello, there are some simple templates in ILLA Builder, you can fork them as your applications.
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/112603073/227829791-8e698ba5-bf2e-4dac-a944-3ecd9ce71cf5.png">
